alf/experience_replayers/replay_buffer.py

```diff
@@ -312,9 +283,9 @@ def get_batch(self, batch_size, batch_length):
                 info = info._replace(
                     importance_weights=self._sum_tree[indices] * avg_weight)
 
-            if self._her_k > 0:
-                result = self._hindsight_relabel(batch_size, batch_length,
-                                                 result, out_env_ids, idx)
+            if self._postprocess_exp_fn:
+                result, info = self._postprocess_exp_fn(
+                    self, result, info, device=self._device)
```

Comment:
No need to pass device. It can be obtained from self

Reply:
Done